tests/TestCli.cpp

```diff
@@ -671,6 +671,23 @@ void TestCli::testInfo()
     QCOMPARE(m_stdout->readLine(), QByteArray("Cipher: AES 256-bit\n"));
     QCOMPARE(m_stdout->readLine(), QByteArray("KDF: AES (6000 rounds)\n"));
     QCOMPARE(m_stdout->readLine(), QByteArray("Recycle bin is enabled.\n"));
+    QVERIFY(m_stdout->readLine().contains(
+        QByteArray("Location: /tmp/testcli"))); // name is randomly generated so just test the path
```

Comment:
Just remove the path portion of this check, it is not a portable test, the path is different on each platform. Alternatively inject the path of the current test database file into the expected output.